addons/l10n_ec/migrations/3.8/post-migrate_update_taxes.py

```diff
@@ -0,0 +1,7 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.account.models.chart_template import update_taxes_from_templates
+
+def migrate(cr, version):
+    # For April 5 Withholding Tax Code Updates
+    update_taxes_from_templates(cr, 'l10n_ec.l10n_ec_ifrs')
```

Comment:
https://github.com/odoo/odoo/blob/saas-16.3/addons/l10n_fr/migrations/2.1/post-migrate_update_taxes.py
the new updating taxes from saas-16.2 onwards